### ISSUE TITLE
[Видалити гурток] User with default (manager) role can delete clubs created by other users'

### ISSUE THREAD

**Issue by yuliiahorak** [ita-social-projects/TeachUA#690]
**Environment:**  Windows 10, Google Chrome 92.0.4515.107
**Reproducible:** always
**Build found:** last commit

**Preconditions**
The user should be logged in the site https://speak-ukrainian.org.ua/dev/

**Steps to reproduce**
1. Click on "Гуртки" button
2. Click on the right corner of the club
3. Click on the setting button
4. Click on "Видалити" -> "Видалити гурток"

**Actual result**
User with default (manager) role can delete clubs created by other users'

https://user-images.githubusercontent.com/82143371/128421589-654e5eff-8d57-480d-8341-b1c790effdd0.mp4

**Expected result**
User with default (manager) role can not delete clubs created by other users'

**User story and test case links**
 "User story #

**Comment by KostiantynStarynets** [comment]
If you do deliting through "Особистий кабінет", then the deletion is successful. If you delete according to the requirements, nothing happens when you click the "Видалити гурток" button, but after refreshing the page, the "Гурток" is deleted.
![Знімок екрана 2023-01-05 225450](https://user-images.githubusercontent.com/117648097/210878907-f1071941-bcfe-41b3-b990-839f6e37780c.jpg)
![Знімок екрана 2023-01-05 225925](https://user-images.githubusercontent.com/117648097/210878911-8c195f71-ea96-42ab-bf2e-96544d15e951.jpg)

**Comment by Vekeryk** [comment]
Yeah, that's a bug. Seams very similar to #680 User cannot delete his Club if there is a comment left. I can add that user also cannot delete club if it has custom logo. There are troubles with club relationship in db.

![image](https://user-images.githubusercontent.com/77672827/210892596-e2f38ab9-03a9-4595-b46f-c771ab239091.png)

About your example. Actually, this club Test 20220822002356 still present in db 
![image](https://user-images.githubusercontent.com/77672827/210893579-83474909-679b-452a-9cf9-313535eddef0.png)
